### PR TITLE
Documentation Update

### LINKED ISSUE #109: documentation update
This comment is no longer true: https://github.com/amiaopensource/vrecord#bmdcapture-update.

This was because bmdtools only has a release point, https://github.com/lu-zero/bmdtools/releases, before the fix to the green/purple issue referenced in documentation. There's now a fork of bmdtools in amiaopensource which adds a release point, https://github.com/amiaopensource/bmdtools/releases, after that point.

### PR DESCRIPTION
**Pull request by savcampbell**
Fixes #109 and #110 and adds CC-BY-V4 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Vrecord Documentation #
-This documentation is up to date as of vrecord v0.7.24 (Released on 2016–10–29)
+This documentation is up to date as of vrecord v0.7.25 (Released on 2016–12–01)
+
+
+## License ###
+
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />Vrecord is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (retokromer) [inline README.md]: please use `https` instead of `http`

**Comment by dericed** [comment]
LGTM

**Comment by retokromer** [comment]
after adding the protocol for secure communication, LGTM

**Comment by savcampbell** [comment]
Should be good to go now. Thanks!

**Pull request merged**